### PR TITLE
Style and lint

### PR DESCRIPTION
**Pull request by TimothyGu**
Fixes most of #35, doesn't use prettier-eslint yet as I don't think that's necessary.

### CODE REVIEW

File: lib/constructs/interface.js [hunk 1]
```diff
@@ -396,7 +397,7 @@ Interface.prototype.generateSymbols = function () {
   if (Object.keys(unscopables).length) {
     this.str += `
       Object.defineProperty(${this.name}.prototype, Symbol.unscopables, {
-        value: ${JSON.stringify(unscopables, null, '  ')},
+        value: ${JSON.stringify(unscopables, null, "  ")},
```
Comment (domenic) [inline lib/constructs/interface.js]: Huh, I never knew you could do that; I always did `2`.

**Comment by TimothyGu** [comment]
I pulled out the test-adding commit. Now https://github.com/jsdom/webidl2js/pull/47/commits/d5677bdcb9491402715c100853ab81b6a573ec13#diff-e80cf17e46910d548417d47d60230718 should be pristine :ok_hand:

**Comment by TimothyGu** [comment]
Note that the stringifier tests didn't work before this PR (the emitted JS isn't valid JS), and I fixed stringifier code generation in the reformat commit, only after it got too hairy to pull it out :( That was why I originally had the tests with the reformat commit. Should be all good now though.

File: lib/constructs/interface.js (hunk 1 above)
Comment (TimothyGu) [inline lib/constructs/interface.js]: Some people prefer tabs :)

**Pull request merged**